terraform/cert-manager.tf

```diff
@@ -8,16 +13,41 @@ resource "kubernetes_namespace" "cert_manager" {
   }
 }
 
+# CRDs have to be installed differently, there is a drama about it in the community.
+# TL;DR: Helm is not yet ready to upgrade CRDs and this can cause an outage.
+# For more info read https://github.com/helm/helm/issues/7735
+resource "null_resource" "cert_manager_crd" {
+  provisioner "local-exec" {
+    working_dir = path.module
+    command     = <<EOF
+for i in `seq 1 10`; do \
```

Comment:
It will try 10 times and sleep for 10 seconds if the k8s API is not up and running

So really this is just a retry mechanism until the k8s API is up and running

Reply:
:+1: